docs/_docs/windows.md

```diff
@@ -7,142 +7,26 @@ While Windows is not an officially-supported platform, it can be used to run Jek
-
- 1. Install a package manager for Windows called [Chocolatey][]
- 2. Install Ruby via Chocolatey: `choco install ruby -y`
- 3. Reopen a command prompt and install Jekyll: `gem install jekyll`
-
-Updates in the infrastructure of Ruby may cause SSL errors when attempting to use `gem install` with versions of the RubyGems package older than 2.6. (The RubyGems package installed via the Chocolatey tool is version 2.3) If you have installed an older version, you can update the RubyGems package using the directions [here][ssl-certificate-update].
-
-[ssl-certificate-update]: http://guides.rubygems.org/ssl-certificate-update/#installing-using-update-packages
-
-
-### Installing *github-pages* via Chocolatey
-
-This section is part of an article written by [Jens Willmer][jwillmerPost]. To follow the instructions you need to have [Chocolatey][] installed on your system. If you already have a version of Ruby installed you need to uninstall it before you can continue.
-
-
-#### Install Ruby and Ruby development kit
-
-Open a command prompt and execute the following commands:
-
- * `choco install ruby -version 2.2.4`
- * `choco install ruby2.devkit` - _needed for compilation of json gem_
-
-
-#### Configure Ruby development kit
-
-The development kit did not set the environment path for Ruby so we need to do it.
-
- * Open command prompt in `C:\tools\DevKit2`
- * Execute `ruby dk.rb init` to create a file called `config.yml`
- * Edit the `config.yml` file and include the path to Ruby `- C:/tools/ruby22`
- * Execute the following command to set the path: `ruby dk.rb install`
+### Install github-pages
 
-
-#### Nokogiri gem installation
-
-This gem is also needed in the github-pages and to get it running on Windows x64 we have to install a few things.
-
-**Note:** In the current [pre release][nokogiriFails] it works out of the box with Windows x64 but this version is not referenced in the github-pages.
-
-```sh
-choco install libxml2 -Source "https://www.nuget.org/api/v2/"
-
-choco install libxslt -Source "https://www.nuget.org/api/v2/"
-
-choco install libiconv -Source "https://www.nuget.org/api/v2/
-
-gem install nokogiri --^
-   --with-xml2-include=C:\Chocolatey\lib\libxml2.2.7.8.7\build\native\include^
-   --with-xml2-lib=C:\Chocolatey\lib\libxml2.redist.2.7.8.7\build\native\bin\v110\x64\Release\dynamic\cdecl^
-   --with-iconv-include=C:\Chocolatey\lib\libiconv.1.14.0.11\build\native\include^
-   --with-iconv-lib=C:\Chocolatey\lib\libiconv.redist.1.14.0.11\build\native\bin\v110\x64\Release\dynamic\cdecl^
-   --with-xslt-include=C:\Chocolatey\lib\libxslt.1.1.28.0\build\native\include^
-   --with-xslt-lib=C:\Chocolatey\lib\libxslt.redist.1.1.28.0\build\native\bin\v110\x64\Release\dynamic
-```
-
-#### Install github-pages
+You need RubyInstaller-2.4 or newer with installed Devkit.
+RubyInstaller versions before 2.4 are not compatible to jekyll!
```

Comment:
This statement is wrong actually..
Jekyll is fully compatible with Ruby 2.3.x installed via RubyInstaller-2.3.3 and *mingw* DevKit..
([AppVeyor log](https://ci.appveyor.com/project/jekyll/jekyll/build/job/5revhriu0irh38lx#L120) for corroboration.)

For a library (as of `v3.8.0`) that [tests fine even on Ruby 2.2.x](https://ci.appveyor.com/project/jekyll/jekyll/build/job/7i6tgblqvign4igv#L122), it would be incoherent for a Quickstart section to state otherwise..

I'm not against *recommending* Ruby Versions actively supported by `RI-2`, but disagree with stating that Jekyll is incompatible with *any version older than Ruby-2.4*..

Reply:
@ashmaroli My mistake - this statement is wrong. I tried `gem install github-pages` with RubyInstaller-2.3. It installs and seems to work. Still there are the SSL issues with those older releases, so I would still recommend RubyInstaller-2.4 and up.

Reply:
`github-pages` and `jekyll` are two different programs.
`github-pages` is a `jekyll` wrapper that has a highly constrained [environment](https://pages.github.com/versions/).

`https://jekyllrb.com/` concerns only about `jekyll`..